app/src/main/res/raw/changelog.yml

```diff
@@ -4,14 +4,14 @@ v32.0-beta1: |
   ended in February, however, there is one last big mega-update I can delight you with before development
   on this app must necessarily wind down:<br/>
   It may not look like a lot, but I basically replaced the whole architecture of the app (#2506, ...)
-  to do this. Note that this required to drop most local data, such as unsynced edits, downloaded
+  to do this. Note that this required dropping most local data, such as unsynced edits, downloaded
   quests and statistics.<br/>Special thanks to @FloEdelmann for helping with the refactor.
   </p>
   <h3>🔙️ Superpowered Undo</h3>
   <ul>
   <li>
-    You can now <b>undo any edit</b> that hasn't been uploaded yet: <br/>
-    Quest answers, split ways, deleted places, hid quests, created notes and note comments (#1029, #2616)
+    You can now <b>undo any edit</b> that hasn't been uploaded yet:<br/>
```

Comment:
Given this and 
> You can now undo deletions of places even after you have already uploaded it

Which of those is true? Or can we not undo splitting, hiding quests, creating and commenting on notes after upload?